docs/reference/flagd-cli/flagd_start.md

```diff
@@ -28,6 +28,7 @@ flagd start [flags]
   -k, --server-key-path string          Server side tls key path
   -d, --socket-path string              Flagd unix socket path. With grpc the evaluations service will become available on this address. With http(s) the grpc-gateway proxy will use this address internally.
   -s, --sources string                  JSON representation of an array of SourceConfig objects. This object contains 2 required fields, uri (string) and provider (string). Documentation for this object: https://flagd.dev/reference/sync-configuration/#source-configuration
+      --stream-deadline duration        Set a server-side deadline for flagd sync and event streams (default 0, means no deadline).
```

Comment:
[issue] The cloud native approach is to allow setting those settings via an environment variable. Hence, this env var will also apply to the providers, as this variable will be set for the whole deployment (most likely). For the providers, this is already specified [here](https://flagd.dev/reference/specifications/providers/#configuration-options). Hence, we should maybe stick to an integer without a time unit, else we have a breaking behavior between providers and flagd

Reply:
I thought this too, but then I was looking and I don't think we actually read any ENV vars for configuring flagd itself - we just use them to configure the providers it seems (or I can't find the code that reads the env vars).

Given this, I think using a duration is better because is more flexible and consistent with our other flagd options with use durations